src/Raven.Studio/wwwroot/Content/css/pages/edit-index.less

```diff
@@ -20,6 +20,15 @@
     .sourcePreview {
         min-height: 400px;
     }
+
+
+    .no-padding-left {
```

Comment:
do we need that? also class name suggests it is something global. It might be misleading in intellisense.

Reply:
1. Moving this class (and _.no-padding-right_) to be global on _styles-common.less_
2. These classes are used in _editIndex.html_ to make the new bootstrap columns  **closer** to each other.  Without this, the items look too far apart.

If I use `col-md-3` instead of `col-md-4`  for the left block then text breaks,
so this was my prefered solution.

![image](https://user-images.githubusercontent.com/17376974/71183961-9c804900-2281-11ea-9b38-55a06e1c627b.png)

Reply:
ok so now we can use no-padding anywhere ;-)